graphql/util.go

```diff
@@ -771,3 +772,116 @@ func GetMyVolumes(user *user.DBUser) ([]restModel.APIVolume, error) {
 	}
 	return apiVolumes, nil
 }
+
+func getManager(ctx context.Context, vol *host.Volume) (cloud.Manager, error) {
```

Comment:
super nit but could you name this something like getEC2Manager? The cloud manager interface we have supports multiple cloud providers, so having EC2 in the name makes it clear this just supports a couple of them